src/Compilers/CSharp/Portable/Lowering/SpillSequenceSpiller.cs

```diff
@@ -469,6 +468,24 @@ private BoundStatement UpdateStatement(BoundSpillSequenceBuilder builder, BoundS
                                                call.Method,
                                                ImmutableArray.Create(Spill(builder, call.Arguments[0]), Spill(builder, call.Arguments[1])));
                         }
+                        else if (call.Method == _F.Compilation.GetWellKnownTypeMember(WellKnownMember.System_String__op_Implicit_ToReadOnlySpanOfChar))
+                        {
+                            Debug.Assert(call.Arguments.Length == 1);
+                            return call.Update([Spill(builder, call.Arguments[0])]);
+                        }
+
+                        goto default;
+
+                    case BoundKind.ObjectCreationExpression:
+                        var objectCreationExpression = (BoundObjectCreationExpression)expression;
+
+                        if (objectCreationExpression.Constructor.OriginalDefinition == _F.Compilation.GetWellKnownTypeMember(WellKnownMember.System_ReadOnlySpan_T__ctor_Reference))
+                        {
+                            Debug.Assert(objectCreationExpression.Arguments.Length == 1);
+                            return objectCreationExpression.UpdateArgumentsAndInitializer([Spill(builder, objectCreationExpression.Arguments[0])],
```

Comment:
>Spill(builder, objectCreationExpression.Arguments[0]) [](http://example.com/codeflow?start=91&length=53)

Since this argument is passed by reference, I think we should pass corresponding argument ref kind. See how ```VisitObjectCreationExpression``` figures out what value to pass.